token-lending/program/src/instruction.rs

```diff
@@ -543,6 +599,7 @@ pub fn init_reserve(
         AccountMeta::new(reserve_collateral_mint_pubkey, false),
         AccountMeta::new(reserve_collateral_supply_pubkey, false),
         AccountMeta::new(reserve_collateral_fees_receiver_pubkey, false),
+        AccountMeta::new(flash_loan_fees_receiver_pubkey, false),
```

Comment:
I thought about reusing the existing fees receiver, however it will be difficult given that the existing ones are for the collateral token while flash loan doesn't requires collateral.

Reply:
Ok, sounds good 👍🏻